paddle/fluid/inference/tensorrt/engine.h

```diff
@@ -538,6 +540,17 @@ class TensorRTEngine {
     }
   }
 
+  void getEngineInfo() {
+#if IS_TRT_VERSION_GE(8200)
+    auto infer_inspector_ = infer_engine_->createEngineInspector();
```

Comment:
Use `std::unique_ptr` for  handling `IEngineInspector*` pointer.

Reply:
Resolved.